### DISCUSSION TITLE
计算机网络 | hengshuai's blog

### DISCUSSION THREAD

**Discussion by giscus** [ihengshuai/blog#95] [Announcements]
# 计算机网络 | hengshuai's blog

计算机网络是指互联的计算机系统之间通过通信设备和通信线路进行数据交换的系统。计算机网络可以分为局域网、城域网、广域网和因特网等不同的类型。计算机网络使用各种协议来实现不同的功能和服务，如经典的TCP/IP协议

https://blog.usword.cn/frontend/computer-network/index.html

**Comment by ihengshuai** [comment]
这部分会总结TCP/IP重要协议
